kafka/conn.py

```diff
@@ -368,7 +372,11 @@ def connect(self):
                 log.debug('%s: creating new socket', self)
                 assert self._sock is None
                 self._sock_afi, self._sock_addr = next_lookup
-                self._sock = socket.socket(self._sock_afi, socket.SOCK_STREAM)
+                if self.config["socks5_proxy"] is not None:
+                    self._socks5_proxy = Socks5Wrapper(self.config["socks5_proxy"], self.afi)
+                    self._sock = self._socks5_proxy.socket(self._sock_afi, socket.SOCK_STREAM)
+                else:
+                    self._sock = socket.socket(self._sock_afi, socket.SOCK_STREAM)
```

Comment:
Not the cleanest way to go about including this functionality, but a feature added is better than a feature removed...